apm-sniffer/apm-sdk-plugin/lettuce-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/lettuce/v5/AsyncCommandMethodInterceptor.java

```diff
@@ -40,6 +42,8 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         String operationName = "Lettuce/" + asyncCommand.getType().name();
         AbstractSpan span = ContextManager.createLocalSpan(operationName + "/onComplete");
         span.setComponent(ComponentsDefine.LETTUCE);
+        Tags.DB_TYPE.set(span, "Redis");
```

Comment:
I think you should create `Tags#CACHE_TYPE`. Redis is not DB from my understanding.

Reply:
Ignore me, I noticed the old codes showing this as a DB.